src/components/lookup/userLookupModal/lookupForm/LookupForm.tsx

```diff
@@ -0,0 +1,116 @@
+import { useCallback, useState } from 'react';
+
+import { ReactComponent as ChevronSVG } from '../../../../assets/images/chevron.svg';
+import { useI18n } from '../../../../contexts/LanguageContext';
+import { Department, User } from '../../../../types/Types';
+import Button from '../../../form/Button';
+import InputField from '../../../form/InputField';
+import '../../supplierLookupModal/lookupForm/LookupForm.css';
+import SVGIcon from '../../../svgIcon/SVGIcon';
+
+interface LookupModalFormProps {
+  handleFilterCriteria: (criteria: User) => void;
+}
+
+const LookupForm: React.FC<LookupModalFormProps> = ({
+  handleFilterCriteria,
+}): JSX.Element => {
+  const initialFilterCriteria: User = {
```

Comment:
Same as in previous PR: anything not influenced by component's specifics stays outside of component's body.